app/client/debug.go

```diff
@@ -0,0 +1,9 @@
+//go:build debug
+
+package main
+
+import (
+	_ "github.com/pokt-network/pocket/app/client/keybase/debug"
+)
+
+// silence is golden
```

Comment:
Much cleaner 👍🏼

Reply:
🤫💯

Reply:
Devil's advocate for newbies coming to this file:

```suggestion
// This file is only here to act as a feature flag based on the presence of the build tag `debug`.
// If that's set, we essentially trigger the `init()` func in `keystore.go` via the anonymous import above.
```

Of course feel free to change the wording as you see fit, I just think it would be good to provide some color on what's happening here.

Wdyt?

Reply:
totally agree, @deblasis i forget sometimes to codify tribal knowledge.